server/repository/src/migrations/v1_06_00/indexes.rs

```diff
@@ -24,6 +24,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         CREATE INDEX index_temperature_log_store_id ON temperature_log (store_id);
         CREATE INDEX index_temperature_log_location_id ON temperature_log (location_id);
         CREATE INDEX index_temperature_log_temperature_breach_id ON temperature_log (temperature_breach_id);
+        CREATE INDEX index_encounter_clinician_id ON encounter (clinician_id);
```

Comment:
Adding deleted indexes back in since these migrations will go into 1.6 while merging is going into 1.7.